contracts/allocators/LUSDAllocator.sol

```diff
@@ -173,7 +173,7 @@ contract LUSDAllocator is OlympusAccessControlled {
                 ISwapRouter.ExactInputParams memory params = ISwapRouter.ExactInputParams({
                     path: abi.encodePacked(address(weth), poolFee, hopTokenAddress, poolFee, lusdTokenAddress),
                     recipient: address(this), //Send LUSD here
-                    deadline: block.timestamp + 25, //25 blocks, at 12 seconds per block is 5 minutes
+                    deadline: block.timestamp + 5 minutes,
```

Comment:
You can just use `block.timestamp` the caller and the swap is in the same block.

I think the point of this is so you can't initiate a swap from the UI that sits in the mempool for days because your gas fee was too low and then you get rekt from a sandwich attack

Reply:
Ahh i see what you mean now.  OK will fix it, don't think this will require a re-deployment of the contract then.